server/player/player.go

```diff
@@ -1279,6 +1279,10 @@ func (p *Player) StartBreaking(pos cube.Pos, face cube.Face) {
 
 		p.breaking.Store(true)
 
+		if p.GameMode().CreativeInventory() {
+			return
+		}
```

Comment:
The positioning of this just looks incorrect. Is there any reason this return happens after the `p.breaking.Store(true)`? And how about the `p.SwingArm()`? Why does that not happen?